### ISSUE TITLE
add pANN scores to cell metadata

### ISSUE THREAD

**Issue by thayerr** [chris-mcginnis-ucsf/DoubletFinder#195]
Hello and thanks for creating this tool!

I have a tricky dataset with cell types of mixed ploidy levels; the real polyploid cells are heavily annotated as doublets. Consequently, I am doing a lot of manual oversight of doublet identification to keep the real polyploid cells while, ideally, still removing doublets with somewhat lower pANN scores. I would like to store pANN scores in my seurat object, rather than immediately classifying as doublet vs singlet, so that downstream in my analysis pipeline I can check whether pANN scores among a suspected doublet cluster are elevated relative to my experimentally-validated, non-polyploid clusters. I imagine this should work something like so:

AddMetaData(Seurat_Object, sweep.res.list$pN_0.25_pK_0.01$pANN, col.name='pANN_scores')
VlnPlot(Seurat_Object, feature='pANN_scores')

However, the AddMetaData call fails with:
Error: cannot add more or fewer cell meta.data information without values being named with cell names.
What is the best way to index and store the pANN scores per cell?

**Comment by thayerr** [comment]
thanks! If anybody out there has worked with mixed-ploidy cell populations, I'm still curious to hear others' experiences.

**Comment by chris-mcginnis-ucsf** [comment]
Yeah seems just like a Seurat syntax issue -- data/experiment sound very cool though!
